libs/portals/my-pages/information/src/module.tsx

```diff
@@ -29,82 +29,87 @@ const UserNotificationsSettings = lazy(() =>
   import('./screens/UserNotifications/UserNotifications'),
 )
 
-const sharedRoutes = (scopes: string[]) => [
-  {
-    name: m.mySettings,
-    path: InformationPaths.SettingsOld,
-    enabled: scopes.includes(UserProfileScope.write),
-    element: <Navigate to={InformationPaths.Settings} replace />,
-  },
-  {
-    name: m.mySettings,
-    path: InformationPaths.Settings,
-    enabled: scopes.includes(UserProfileScope.write),
-    element: <UserProfileSettings />,
-  },
-  {
-    name: 'Notifications',
-    path: InformationPaths.Notifications,
-    enabled: scopes.includes(DocumentsScope.main),
-    element: <Notifications />,
-  },
-]
+const sharedRoutes = (scopes: string[], isCompany = false) => {
+  const isSettingsEnabled =
+    scopes.includes(UserProfileScope.read) &&
```

Comment:
@GunnlaugurG you will maybe address this when the final conclusion of scope access will be determined.